graphql/resolvers.go

```diff
@@ -1864,6 +1865,24 @@ func (r *mutationResolver) RestartTask(ctx context.Context, taskID string) (*res
 	return apiTask, err
 }
 
+func (r *mutationResolver) MoveIssueToSuspectedIssue(ctx context.Context, annotationID string, apiIssue restModel.APIIssueLink) (bool, error) {
```

Comment:
having 1 mutation + an additional parameter to handle the use cases for these two mutations could result in less code (generated and assignments) in the frontend